ls1intum/Artemis
# [Bugfix] Fix NPE in getRepository of RepositoryProgrammingExerciseParticipationResource

### Checklist

### Motivation and Context


Should fix Sentry issue: https://sentry.io/organizations/ls1intum/issues/1948174164/?environment=prod&project=1440029&query=is%3Aunresolved+assigned%3Ame&statsPeriod=14d

### Description

- Extended the check as it is only relevant for Student Participations

### Steps for Testing


1. Log in to Artemis as Instructor
2. Navigate to Course Administration
3. Go to a Programming Exercise and click on `Edit in editor`
4. Switch between Template Repo and Solution Repo
5. No error should be thrown

NOTE: I was not able to reproduce the issue by doing this test, but if you check the Sentry issue the error is thrown in the GET Call:
![image](https://user-images.githubusercontent.com/41108487/99880591-a9858500-2c14-11eb-82bb-be32599872ec.png)
@krusche checked the participationId and it was a `SolutionParticipation`, so it can be just thrown in the scenario as noted above.

If anyone has an idea how we can reproduce this bug, let me know :)
### Test Coverage



Nothing changed

### Screenshots



![image](https://user-images.githubusercontent.com/41108487/99880715-5b24b600-2c15-11eb-81a2-2b3085e59040.png)
